datalake/src/main/java/com/sequenceiq/datalake/service/sdx/status/SdxStatusService.java

```diff
@@ -110,6 +110,20 @@ public void setStatusForDatalakeAndNotify(DatalakeStatusEnum status, ResourceEve
         eventSenderService.sendEventAndNotification(sdxCluster, event, messageArgs);
     }
 
+    public void setStatusForDatalakeAndNotify(DatalakeStatusEnum status, ResourceEvent event, Collection<?> messageArgs, String statusReason,
+            SdxCluster sdxCluster, String userId) {
```

Comment:
In the statusChecker job you are passing an accountId, here you are referring to the last param as userId?
Which is it then?

Also if it's the account ID, you can get that info fro the SdxCluster/crn object.

Reply:
It is accountId I will make this clear. And using the accound ID from the sdx cluster object is exactly what I did in this change.

Reply:
Fixed